packages/client/src/user/userReducer.ts

```diff
@@ -67,7 +67,8 @@ const initialState: IUserFormState = {
   loadingRoles: false,
   submissionError: false,
   userAuditForm,
-  systemRoleMap: {}
+  systemRoleMap: {},
+  updateToBeConfirmed: false
```

Comment:
Would it make sense if `updateToBeConfirmed` would be named `userDetailsChanged` or similar?